### PR TITLE
refactor(console): improve create app form ux

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

- add error message for application type selection
- add focus effect for `<Radio>`
- add key press handler for `<Radio>`

## Testing

image

also tested with keyboard events

@logto-io/eng

### CODE REVIEW

File: packages/console/src/pages/Applications/components/CreateForm/index.tsx [hunk 1]
```diff
@@ -61,6 +66,9 @@ const CreateForm = ({ onClose }: Props) => {
               </Radio>
             ))}
           </RadioGroup>
+          {errors.type?.type === 'required' && (
```
Comment (simeng-li) [inline packages/console/src/pages/Applications/components/CreateForm/index.tsx:69]: `errors.type?.type` ?

File: packages/console/src/components/RadioGroup/Radio.tsx [hunk 2]
```diff
@@ -22,12 +22,27 @@ export type Props = {
...
   isChecked?: boolean;
   onClick?: () => void;
+  tabIndex?: number;
 };
 
-const Radio = ({ value, title, name, children, isChecked, onClick }: Props) => {
+const Radio = ({ value, title, name, children, isChecked, onClick, tabIndex }: Props) => {
+  const handleKeyPress: KeyboardEventHandler<HTMLDivElement> = useCallback(
+    ({ key }) => {
+      if ([' ', '\n'].includes(key)) {
```
Comment (simeng-li) [inline packages/console/src/components/RadioGroup/Radio.tsx:31]: key should be '  ' and 'Enter'? nit: keycode might be more precise.

**Review by simeng-li — COMMENTED** [review]
overall LGTM

File: packages/console/src/components/RadioGroup/Radio.tsx (hunk 2 above)
Comment (gao-sun) [reply packages/console/src/components/RadioGroup/Radio.tsx:31]: `keyCode` has been [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode)

File: packages/console/src/pages/Applications/components/CreateForm/index.tsx (hunk 1 above)
Comment (gao-sun) [reply packages/console/src/pages/Applications/components/CreateForm/index.tsx:69]: yes, the first type is a field name and the second one is error type, see https://react-hook-form.com/get-started#Handleerrors

File: packages/console/src/components/RadioGroup/Radio.tsx (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/components/RadioGroup/Radio.tsx:31]: It doesn't work on my machine.
`Tab` to focus the radio successfully but failed when I press `Enter` to select.

**Review by xiaoyijun — APPROVED** [review]
LGTM

File: packages/console/src/components/RadioGroup/Radio.tsx (hunk 2 above)
Comment (gao-sun) [reply packages/console/src/components/RadioGroup/Radio.tsx]: fixed, ptal

**Pull request merged**